extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonUtils.java

```diff
@@ -367,6 +368,11 @@ public static String signJwtWithKey(OidcCommonConfig oidcConfig, String tokenReq
         }
     }
 
+    @SuppressWarnings({ "unchecked", "rawtypes" })
```

Comment:
This method is only introduced so you don't have to annotate signJwtWithKey with SuppressWarnings, right?

Reply:
@lordvlad Yes, it is guaranteed Quarkus Config will init an empty or non empty Map with string value. When dealing directly with the JWT build api, Map can contains complex claims like arrays or objects